device/mellanox/x86_64-nvidia_sn5600-r0/Mellanox-SN5600-C256/port_config.ini

```diff
@@ -0,0 +1,275 @@
+Ethernet449    449                                    etp57b    57       100000   rs    off
+Ethernet450    450                                    etp57c    57       100000   rs    off
+Ethernet451    451                                    etp57d    57       100000   rs    off
+Ethernet452    452                                    etp57e    57       100000   rs    off
+Ethernet453    453                                    etp57f    57       100000   rs    off
+Ethernet454    454                                    etp57g    57       100000   rs    off
+Ethernet455    455                                    etp57h    57       100000   rs    off
+Ethernet464    464                                    etp59a    59       100000   rs    off
+Ethernet465    465                                    etp59b    59       100000   rs    off
+Ethernet466    466                                    etp59c    59       100000   rs    off
+Ethernet467    467                                    etp59d    59       100000   rs    off
+Ethernet468    468                                    etp59e    59       100000   rs    off
+Ethernet469    469                                    etp59f    59       100000   rs    off
+Ethernet470    470                                    etp59g    59       100000   rs    off
+Ethernet471    471                                    etp59h    59       100000   rs    off
+Ethernet480    480                                    etp61a    61       100000   rs    off
+Ethernet481    481                                    etp61b    61       100000   rs    off
+Ethernet482    482                                    etp61c    61       100000   rs    off
+Ethernet483    483                                    etp61d    61       100000   rs    off
+Ethernet484    484                                    etp61e    61       100000   rs    off
+Ethernet485    485                                    etp61f    61       100000   rs    off
+Ethernet486    486                                    etp61g    61       100000   rs    off
+Ethernet487    487                                    etp61h    61       100000   rs    off
+Ethernet496    496                                    etp63a    63       100000   rs    off
+Ethernet497    497                                    etp63b    63       100000   rs    off
+Ethernet498    498                                    etp63c    63       100000   rs    off
+Ethernet499    499                                    etp63d    63       100000   rs    off
+Ethernet500    500                                    etp63e    63       100000   rs    off
+Ethernet501    501                                    etp63f    63       100000   rs    off
+Ethernet502    502                                    etp63g    63       100000   rs    off
+Ethernet503    503                                    etp63h    63       100000   rs    off
+Ethernet512    512                                    etp65     65        25000   rs
```

Comment:
because the profile contains 1 25G port, do you mind to rename the folder into ...-C256X1?

Reply:
X is the letter for 25G.